src/components/planning/ReasonCheckboxes.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import {FormField} from "../form-components/FormComponents"
+import styled from "styled-components"
+import {isRequired} from "../form-components/validators/isRequired"
+
+const REASONS = [
+  'Bed en breakfast 2019',
+  'Burgwallenproject Oudezijde',
+  'Corpo-rico',
+  'Digital toezicht Safari',
+  'Digital toezicht Zebra',
+  'Haarlemmerbuurt',
+  'Hotline',
+  'Mystery Guest',
+  'Project Andes',
+  'Project Jordaan',
+  'Project Lobith',
+  'Project Sahara',
+  'Safari',
+  'Safari 2015',
+  'Sahara Adams Suites',
+  'Sahara hele woning',
+  'Sahara meer dan 4',
+  'Sahara Recensies',
+  'Sahara veel adv',
+  'Social Media 2019',
+  'Woonschip (woonboot)',
+  'Zebra'
+]
```

Comment:
These options are being loaded from the backend. Should not be hardcoded.

Reply:
Done